src/test/java/core/basesyntax/service/implementations/ReadServiceImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.implementations;
+
+import core.basesyntax.service.inerfaces.ReadService;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReadServiceImplTest {
+    private static ReadService read;
+    private static final String RIGHT_DATA_ROUTE
+            = "src/test/java/core/basesyntax/resources/RightInput.cvs";

```

Comment:
```suggestion
            = "src/test/resources/RightInput.cvs";
```
так само для інших файлів

Reply:
Can't read from file src/test/resources/RightInput.cvs